### ISSUE TITLE
Remapping unmapped params to less unreadable names in decompiled Minecraft code

### ISSUE THREAD

**Issue by Niko-sk2x** [DemonWav/MinecraftDevIntelliJ#657]
I suggested it on discord initially.

In a lot of Minecraft code is unmapped, and with more uses of lambdas (and lambda parameters not being deobfuscated) it would be very useful to automatically change them to something more human friendly, like a name derived from variable type, for example:
```
blockPos, bloskPos2, world, f, f2, f3
```
As such names can be much easier to follow than `p_637293_2_` and `p_832481_1_` and similar.

**Comment by Frontear** [comment]
I don't see how this is MinecraftDev's problem. This is strictly a problem for the modding environment you are using. MinecraftDev isn't going to help with mapping, it's only here to make programming easier, and this goes out of the scope.

Mappings are the individual responsibility of the environment, Forge, Fabric, Sponge whatever.

**Comment by Niko-sk2x** [comment]
I suggested it on discord and got a reply that I should make an issue on github. The problem with it being done in forge is that it will never be done because obfuscated names are in the current format intentionally. And this is a format that happens to make the code very hard to read. 

Still in the meantime I made myself a tool to generate some less unreadable mapping names so for me it's no longer needed. And since so far no-one else seems to need or want this I'm closing it.
